Formula/nginx-unit.rb

```diff
@@ -0,0 +1,267 @@
+class NginxUnit < Formula
+  desc "Dynamic web and application server for multiple languages"
+  homepage "https://unit.nginx.org/"
+  url "https://unit.nginx.org/download/unit-1.1.tar.gz"
+  sha256 "7c66365f5ea87e7e8903bac9d639676947fe2ab7aa799ce35defc426b3409ee0"
+
+  depends_on "openssl"
+  depends_on "pcre"
+
+  def install
+    openssl = Formula["openssl"]
+    pcre = Formula["pcre"]
+
+    system "./configure",
+      "--prefix=#{prefix}",
+      "--bindir=#{bin}",
+      "--sbindir=#{bin}",
+      # This tells the unit daemon where it can load modules from. Configuring
+      # the location at runtime is painful. We want this to be in the
+      # $HOMEBREW_PREFIX so that other formulae can link in additional modules. 
+      "--modules=#{HOMEBREW_PREFIX/"lib/unit"}",
```

Comment:
It's a nit but writing it this way is much nicer to look at:

```diff
diff --git a/Formula/nginx-unit.rb b/Formula/nginx-unit.rb
index b43c12a700..5a51c4fe2d 100644
--- a/Formula/nginx-unit.rb
+++ b/Formula/nginx-unit.rb
@@ -18,11 +18,11 @@ class NginxUnit < Formula
       # This tells the unit daemon where it can load modules from. Configuring
       # the location at runtime is painful. We want this to be in the
       # $HOMEBREW_PREFIX so that other formulae can link in additional modules.
-      "--modules=#{HOMEBREW_PREFIX/"lib/unit"}",
-      "--state=#{var/"unit"}",
-      "--log=#{var/"log/unit.log"}",
-      "--pid=#{var/"run/unit.pid"}",
-      "--control=unix:#{var/"run/unit.control.sock"}",
+      "--modules=#{HOMEBREW_PREFIX}/lib/unit",
+      "--state=#{var}/unit",
+      "--log=#{var}/log/unit.log",
+      "--pid=#{var}/run/unit.pid",
+      "--control=unix:#{var}/run/unit.control.sock",
       "--cc-opt=-I#{pcre.opt_include} -I#{openssl.opt_include}",
       "--ld-opt=-L#{pcre.opt_lib} -L#{openssl.opt_lib}"
 
@@ -62,7 +62,7 @@ class NginxUnit < Formula
         curl --unix-socket #{var}/run/unit.control.sock http://localhost/
 
       Modules are available for system perl, python and ruby. The system php doesn't support nginx-unit. You can install additional modules into:
-        #{HOMEBREW_PREFIX/"lib/unit"}
+        #{HOMEBREW_PREFIX}/lib/unit
     EOS
   end
```

Reply:
I had it that way before, will switch it back.